spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,6 @@
+# Needed for facter to fetch facts used by the postgresql module
+if fact('os.name') == 'Ubuntu' and fact('os.release.full') == '18.04' {
```

Comment:
Instead of checking for ubuntu 18.04 we can check for Facter < 4, that way we won't install lsb-release on Puppet 7 which is not needed.

```ruby
if versioncmp($facts['facterversion'], '4.0.0') < 0
```